### ISSUE TITLE
wrong version in transfer.sh-1.0.0.tar.gz and git version

### ISSUE THREAD

**Issue by truatpasteurdotfr** [dutchcoders/transfer.sh#238]
cosmetic issue: the version printed when doing `transfersh --help` and `transfersh --version` are inconsistent.

```
$ grep 'Version =' transfer.sh-1.0.0/cmd/cmd.go
transfer.sh-1.0.0/cmd/cmd.go:var Version = "0.1"
```
while transfersh --version always print 0.0.0

**Comment by aspacca** [comment]
will fix, thanks

**Comment by aspacca** [comment]
@truatpasteurdotfr fixed in #247, it will affect from version 1.1.0
